src/Airship/Internal/Helpers.hs

```diff
@@ -8,24 +8,32 @@ module Airship.Internal.Helpers where
 #if __GLASGOW_HASKELL__ < 710
 import           Control.Applicative
 #endif
-import           Data.ByteString     (ByteString)
+import           Data.ByteString               (ByteString)
+import qualified Data.ByteString.Lazy.Internal as LazyBS
```

Comment:
Does 'internal' convey something that means we should not be importing/using it?